test/ci.py

```diff
@@ -159,11 +160,19 @@ def test_parse_vsc_ci_cfg(self):
         error_msg = "Unknown key in vsc-ci.ini: unknown_key"
         self.assertErrorRegex(ValueError, error_msg, parse_vsc_ci_cfg)
 
-        vsc_ini_txt = '\n'.join('%s=1' % key for key in default.keys())
-        vsc_ini_txt = re.sub('additional_test_commands=1', 'additional_test_commands=./more_tests.sh', vsc_ini_txt)
-        self.write_vsc_ci_ini(vsc_ini_txt)
+        vsc_ini_txt_lines = []
+        for key in default.keys():
+            # special treatment needed for non-boolean keys like additional_test_commands
+            if default[key] is None:
+                vsc_ini_txt_lines.append('%s=foo' % key)
+            else:
+                vsc_ini_txt_lines.append('%s=1' % key)
```

Comment:
So, 1 maps to True?

Reply:
Yes, those are boolean configuration settings.